### PR TITLE
Magda/fix accuracy

### PR DESCRIPTION
**Pull request by mSkarbinska**
example result of get all wordsets endpoint:
```
[{
  id: 5,
  english: 'food 2',
  polish: 'żywność 2',
  category: 'food',
  difficulty: 2,
  locked: true,
  accuracy: -1,
  depends_on: [
    {
      id: 1,
      english: 'food',
      polish: 'żywność',
      category: 'food',
      difficulty: 1
    }
  ]
}
]
```
accuracy = -1 means it was never played

### CODE REVIEW

**Comment by mSkarbinska** [comment]
obraz

Example result:
```
{
  '1': 0,
  '2': 0,
  '3': 0,
  '4': 0,
  '5': 0,
  '6': 0,
  '7': 2,
  '8': 4,
  '9': 0,
  '10': 0,
  '11': 0,
  '12': 0,
  '13': 0,
  '14': 0,
  '15': 0,
  '16': 0,
  '17': 0,
  '18': 0,
  '19': 0,
  '20': 0,
  '21': 0,
  '22': 0,
  '23': 0,
  '24': 0,
  '25': 0,
  '26': 0,
  '27': 0,
  '28': 0,
  '29': 0,
  '30': 0,
  '31': 0
}
```

File: api/models.py [hunk 1]
```diff
@@ -18,42 +18,6 @@ def __str__(self) -> str:
         return self.english
 
 
-class TranslationUserAccuracyCounter(models.Model):
```
Comment (MichalGniadek) [inline api/models.py:21]: Doesn't this require a migration?

**Review by MichalGniadek — APPROVED** [review]

File: api/models.py (hunk 1 above)
Comment (mSkarbinska) [reply api/models.py:21]: i think it works without it because it is unused, though existent in db, will migrate

**Comment by mSkarbinska** [comment]
```
{ longest_streak: 2, 
start_date: '2023-12-07', 
end_date: '2023-12-08' }
```
obraz

endpoint: `/longest-streak/`

**Comment by mSkarbinska** [comment]
to do: remove repetition in code

**Review by ahurbol — CHANGES_REQUESTED** [review]
Accuracy is working a bit poorly:

-  Let's count the number of points for each set, and after reaching, for example, 1000 points, let the next set unlock instead of relying on accuracy.
- Also, instead of returning accuracy for /wordset/ endpoint, it should return points.


Statistics:

-  All statistics endpoints should start with /statistics/ (for example  /statistics/calendar/,  /statistics/streak/).
- To do: Current streak.
- To do: Total points from the game (or all games).
- If there is an option: for all games option in the longest-streak (same to current streak and total points), can the body be {game: "all_games"} instead of bypassing the body?

File: backend/urls.py [hunk 2]
```diff
@@ -37,8 +36,11 @@
 
 urlpatterns = [
     path("", include(router.urls)),
-    path("scoreboard/", views.get_scoreboard),
-    path("user-statistics/", views.get_user_statistics),
+    path("statistics/scoreboard/", views.get_scoreboard),
```
Comment (ahurbol) [inline backend/urls.py:39]: scoreboard without statistics prefix, because it is other functionality

**Pull request merged**